android/app/src/main/java/org/fossasia/openevent/activities/TrackSessionsActivity.java

```diff
@@ -134,6 +143,33 @@ private void loadData() {
 
             mSessions.clear();
             mSessions.addAll(track.getSessions().sort("startsAt"));
+
+//            finding upcoming and ongoing sessions
+            Date current;
+            int countUpcoming = 0;
+            int countOngoing = 0;
+            for (Session trackSession : mSessions) {
+                flag = 0;
+                Date start = new Date();
+                Date end = new Date();
+                try {
+                    start = DateUtils.getDate(trackSession.getStartsAt());
+                    end = DateUtils.getDate((trackSession.getEndsAt()));
+                } catch (ParseException e) {
+                    e.printStackTrace();
+                }
+                current = new Date();
+                if (((start.before(current)) || (start.equals(current))) && ((end.after(current)) || (end.equals(current)))) {
+                    ongoingPosition = countUpcoming;
+                    break;
+                } else if (start.after(current)) {
+                    upcomingPosition = countOngoing;
+                    break;
+
+                } else flag += 1;
+                countUpcoming += 1;
+                countOngoing += 1;
```

Comment:
Yes, that too gives an error. Could you have a look please?
<img width="1191" alt="screen shot 2017-07-19 at 2 50 41 pm" src="https://user-images.githubusercontent.com/21010060/28359996-b77e827e-6c91-11e7-8cb3-84584b6509b1.png">